libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/infopanel/InfoPanelHeaderArrivalBinder.kt

```diff
@@ -0,0 +1,30 @@
+package com.mapbox.navigation.dropin.binder.infopanel
+
+import android.transition.Scene
+import android.transition.TransitionManager
+import android.view.ViewGroup
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver
+import com.mapbox.navigation.dropin.NavigationViewContext
+import com.mapbox.navigation.dropin.R
+import com.mapbox.navigation.dropin.databinding.MapboxInfoPanelHeaderArrivalLayoutBinding
+import com.mapbox.navigation.dropin.internal.endNavigationButtonComponent
+import com.mapbox.navigation.ui.base.lifecycle.UIBinder
+
+@ExperimentalPreviewMapboxNavigationAPI
+internal class InfoPanelHeaderArrivalBinder(
+    private val context: NavigationViewContext
+) : UIBinder {
+
+    override fun bind(viewGroup: ViewGroup): MapboxNavigationObserver {
+        val scene = Scene.getSceneForLayout(
+            viewGroup,
+            R.layout.mapbox_info_panel_header_arrival_layout,
+            viewGroup.context
+        )
+        TransitionManager.go(scene)
+        val binding = MapboxInfoPanelHeaderArrivalLayoutBinding.bind(viewGroup)
+
+        return context.endNavigationButtonComponent(binding.endNavigation)
```

Comment:
We also show the text `Arrived` in the header upon arrival. At the moment there is no component for this text except for the fact that it's only added in the XML `mapbox_info_panel_header_arrival_layout`. Similar to other components we should have a dedicated `ArrivalTextComponent` whose sole purpose as per current design will be to show the text `Arrived` but later can be modified as per mutating designs. This would also allow us to inject different text appearance styles to the text `Arrived`.